model/task/task_test.go

```diff
@@ -1273,41 +1273,88 @@ func TestGetTaskIdBetweenIds(t *testing.T) {
 		assert.NoError(task.Insert())
 		tasks = append(tasks, task)
 	}
-	t10, err := GetMidwayTask(tasks[0], tasks[19])
+	t10, err := FindMidwayTask(tasks[0], tasks[19])
 	assert.NoError(err)
-	assert.NotNil(t10)
-	assert.Equal(t10.RevisionOrderNumber, 10)
+	require.NotNil(t, t10)
+	assert.Equal(10, t10.RevisionOrderNumber)
 
-	t5, err := GetMidwayTask(tasks[0], tasks[9])
+	t5, err := FindMidwayTask(tasks[0], tasks[9])
 	assert.NoError(err)
-	assert.NotNil(t5)
-	assert.Equal(t5.RevisionOrderNumber, 5)
+	require.NotNil(t, t5)
+	assert.Equal(5, t5.RevisionOrderNumber, 5)
 
-	t15, err := GetMidwayTask(tasks[10], tasks[19])
+	t15, err := FindMidwayTask(tasks[10], tasks[19])
 	assert.NoError(err)
-	assert.NotNil(t15)
-	assert.Equal(t15.RevisionOrderNumber, 15)
+	require.NotNil(t, t15)
+	assert.Equal(15, t15.RevisionOrderNumber)
 
-	t19, err := GetMidwayTask(tasks[17], tasks[19])
+	t19, err := FindMidwayTask(tasks[17], tasks[19])
 	assert.NoError(err)
-	assert.NotNil(t19)
-	assert.Equal(t19.RevisionOrderNumber, 19)
+	require.NotNil(t, t19)
+	assert.Equal(19, t19.RevisionOrderNumber)
 
-	t4, err := GetMidwayTask(tasks[6], tasks[0])
+	t4, err := FindMidwayTask(tasks[6], tasks[0])
 	assert.NoError(err)
-	assert.NotNil(t4)
-	assert.Equal(t4.RevisionOrderNumber, 4)
+	require.NotNil(t, t4)
+	assert.Equal(4, t4.RevisionOrderNumber)
 
-	t12, err := GetMidwayTask(tasks[11], tasks[11])
+	t12, err := FindMidwayTask(tasks[11], tasks[11])
 	assert.NoError(err)
-	assert.NotNil(t12)
-	assert.Equal(t12.RevisionOrderNumber, 12)
+	require.NotNil(t, t12)
+	assert.Equal(12, t12.RevisionOrderNumber)
 
-	t16, err := GetMidwayTask(tasks[15], tasks[16])
+	t16, err := FindMidwayTask(tasks[15], tasks[16])
 	assert.NoError(err)
-	assert.NotNil(t16)
-	assert.Equal(t16.RevisionOrderNumber, 16)
-
+	require.NotNil(t, t16)
+	assert.Equal(16, t16.RevisionOrderNumber)
+
+	otherDisplayName := Task{
+		Id:           "otherTask",
```

Comment:
This is gonna cause a write error when inserting inot the  DB since it's invalid to insert multiple docs with the same `_id`. You can use `utility.RandomString()` if you just want a random name, or just fill in some ID without any other dups in the test.